pkg/processor/statefulset/statefulset.go

```diff
@@ -0,0 +1,138 @@
+package statefulset
```

Comment:
Please add statefulset sample to `test_data/sample-app.yaml`
And regenerate examples in /examples directory by running:

```shell
cat test_data/sample-app.yaml | go run ./cmd/helmify examples/app
```